DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -381,6 +387,63 @@ public override void Initialize()
             AddChild(btnPickRandomMap);
         }
 
+        private void InitializeGameOptionsPanel()
+        {
+            GameOptionsPanel = new XNAPanel(WindowManager);
+            GameOptionsPanel.Name = "GameOptionsPanel";
```

Comment:
Use `nameof`. `GameOptionsPanel.Name = nameof(GameOptionsPanel);`

Reply:
Will do. This was actually moved as-is from a parent class.